<!-- -*- mode: XML -*- -->
<launch>
    <group ns="komodo_1">
		<include file="$(find komodo_arm_moveit_config)/launch/move_group.launch"/>
    </group>

    <node name="move_arm" pkg="pluto" type="moveArm.py"  args="" respawn="true" />
    <node name="movement" pkg="pluto" type="movement.py" args="" respawn="true" />
    <node name="detector" pkg="pluto" type="detector.py" args="" respawn="true" />
    <!--<node name="mickey"   pkg="pluto" type="mickey.py"   args="" respawn="true" />-->
  
	
</launch>

